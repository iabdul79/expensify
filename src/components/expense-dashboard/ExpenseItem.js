import React from 'react'

export default (props) => {
  const {description, amount, createdAt} = props.expense;
  return (
    <div>
      <h2>{description}</h2>
      <p>{amount} - {createdAt}</p>
  </div>
  )
}