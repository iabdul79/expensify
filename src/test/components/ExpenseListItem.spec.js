import React from 'react'
import {shallow} from 'enzyme'
import ExpenseItem from '../../components/expense-dashboard/ExpenseItem'
import expenses from '../fixtures/expenses'

test('should show expense item details for a passed item', () => {
  const wrapper = shallow(<ExpenseItem expense={expenses[2]} />)
  expect(wrapper).toMatchSnapshot()
})