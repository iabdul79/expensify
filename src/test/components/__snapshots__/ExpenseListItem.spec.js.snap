// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should show expense item details for a passed item 1`] = `
<div>
  <[object Object]
    to="/edit/3"
  >
    <h2>
      Credit card
    </h2>
  </[object Object]>
  <p>
    12100
     - 
    345600000
  </p>
</div>
`;
